developers_chamber/scripts/qa.py

```diff
@@ -0,0 +1,246 @@
+import re
+import subprocess
+
+from developers_chamber.scripts import cli
+from developers_chamber.utils import print_bold, print_error, print_heading, print_success
+from git import Repo
+
+
+class QAError(Exception):
```

Comment:
There should be only Click bindings in this module. Everything else belongs to `<something>_utils.py`. Either use some existing modules if applicable or just create your own.

Reply:
Yeah, I was thinking about splitting the file.